src/radical/pilot/agent/agent_0.py

```diff
@@ -95,6 +103,8 @@ def __init__(self, cfg, session):
         rpu.Worker.__init__(self, self._cfg, session)
 
         self.register_subscriber(rpc.CONTROL_PUBSUB, self._check_control)
+        # TODO do I need to pass pilot sandbox
```

Comment:
No - the callback  runs in a separate thread, but has access to the class state, so you don't need to pass anything.

Reply:
ok